curriculum/challenges/english/25-front-end-development/quiz-javascript-dates/66edd3711bb9f7efa73aef91.md

```diff
@@ -17,439 +17,455 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+The `Date` object in javascript is used to?
 
 #### --distractors--
 
-Placeholder distractor 1
+Work with just dates only.
 
 ---
 
-Placeholder distractor 2
+Work with just time only.
 
 ---
 
-Placeholder distractor 3
+None of the above.
 
 #### --answer--
 
-Placeholder answer
+Work with both dates and time.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which code below creates a new `Date` object instance?
 
 #### --distractors--
 
-Placeholder distractor 1
+`Date.new()`
 
 ---
 
-Placeholder distractor 2
+`Date.fetch()`
 
 ---
 
-Placeholder distractor 3
+`new.Date()`
 
 #### --answer--
 
-Placeholder answer
+`new Date()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How does `Date.now()` work?
 
 #### --distractors--
 
-Placeholder distractor 1
+It returns the current time in nanoseconds since July 4, 1776.
 
 ---
 
-Placeholder distractor 2
+It returns the previous time in milliseconds since January 1, 1974.
 
 ---
 
-Placeholder distractor 3
+It returns the current date in minutes since January 1, 1970.
 
 #### --answer--
 
-Placeholder answer
+It returns the current time in milliseconds since January 1, 1970.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+To get the current year we use which instance method of the `Date` object?
 
 #### --distractors--
 
-Placeholder distractor 1
+`getNewYear()`
 
 ---
 
-Placeholder distractor 2
+`getLastYear()`
 
 ---
 
-Placeholder distractor 3
+`getLeapYear()`
 
 #### --answer--
 
-Placeholder answer
+`getFullYear()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+To get the current month (0 - 11) we use which instance method of the `Date` object?
 
 #### --distractors--
 
-Placeholder distractor 1
+`getCalender()`
 
 ---
 
-Placeholder distractor 2
+`fetchMonth()`
 
 ---
 
-Placeholder distractor 3
+`get.Month()`
 
 #### --answer--
 
-Placeholder answer
+`getMonth()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+The built-in JavaScript method that allows us to format a date as a string is?
 
 #### --distractors--
 
-Placeholder distractor 1
+`getMonth()`
 
 ---
 
-Placeholder distractor 2
+`formatDate()`
 
 ---
 
-Placeholder distractor 3
+`toStingy()`
 
 #### --answer--
 
-Placeholder answer
+`toString()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What would output `console.log(new Date().getFullYear());`, if it would be the year `2022`?
```

Comment:
```suggestion
What would output `console.log(new Date().getFullYear());`, if it would be June 12th 2022?
```

so that the June answer makes sense. One of the other two distractors could maybe be some other version of this date?

Reply:
Yes I feel that would make more sense than just giving out the answer. Let me quickly run the changes.

Reply:
Do this distractors make sense, I hope it was in line with your suggestion ?

### --question--

#### --text--

What would output `console.log(new Date().getFullYear());`, if it would be June 12th 2022?

#### --distractors--

`"June 12th 2022"`

---

`2023`

---

`06/12/2022`

#### --answer--

`2022`

Reply:
They make sense even before. But if one of the distractor it's a month it makes sense to have it in the question date too

Reply:
Oh Yes I see